crates/storage/src/lazy/mapping.rs

```diff
@@ -138,7 +138,7 @@ where
     ///
     /// # Panics
     ///
-    /// Traps if the encoded `value` doesn't fit into the static buffer.
+    /// Traps if the the encoded `key` or `value` doesn't fit into the static buffer.
```

Comment:
```suggestion
    /// Traps if the the encoded `key` and `value` doesn't fit into the static buffer.
```

Good catch! However they are encoded together so the total size must fit to not trap, WDYT?